Add /* when routes really need to match the whole path.

mguillet opened the pull request:
Example:
mainRouter.route("/static") -> mainRouter.route("/static/")
The documentation doesn't reflect what's in the vertx-example repo code.

purplefox commented:
Hi Michel,

The asciidoc is generated from the text in the package-info.java file. Could you make the changes in there?

Also.. can I check if you have signed the Eclipse CLA?

mguillet commented:
Tim,

The fixes were on the examples, so I've fixed the `Examples.java` file.

I've rewrote my commit with a proper author and signed it off. I think I've signed the CLA a long time ago…

One question though: shall I include the result of the documentation generation in the commit or just changes to the master file?

purplefox commented:
Please include any changes whether generated or not :)

mguillet commented:
All right, this should be better now: Java, Groovy and JS docs are now updated.

Is this OK for you?

purplefox commented:
Michel,

The PR appears to have conflicts now. I think you need to rebase.

mguillet commented:
:-) all good now

purplefox commented:
Thanks
